eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -34,7 +34,7 @@ param()
 
 # Since we're skipping Management for the moment, only look for files with certain parents. These
 # limitations will vanish once Management track is updated.
-$ValidParents = ("azure-sdk-parent", "azure-client-sdk-parent", "azure-client-sdk-parent-v2", "azure-data-sdk-parent", "azure-perf-test-parent", "azure-cosmos-spark_3_2-12", "io.clientcore:clientcore-parent")
+$ValidParents = ("azure-sdk-parent", "azure-client-sdk-parent", "azure-data-sdk-parent", "azure-perf-test-parent", "azure-cosmos-spark_3_2-12", "io.clientcore:clientcore-parent")
```

Comment:
Should we add groupIds here too for better identification com.azure:azure-client-sdk-parent', 'com.azure.v2:azure-client-sdk-parent'. I see we are doing that for clientcore-parent anyway?

Reply:
I'll investigate this as a follow-up. There are likely going to be many more EngSys changes needed along the way.